docs/manuals/source/TasksAndConcepts/Plugins/FileDaemonPlugins/OVirtPlugin.rst.inc

```diff
@@ -3,9 +3,22 @@
 oVirt Plugin
 ~~~~~~~~~~~~
 
+.. warning::
+
+   The oVirt Plugin has been removed in :sinceVersion:`23: oVirt Plugin` because
+   the plugin behaves unstable with Python 3.
+
+   As Red Hat already has announced that Red Hat Virtualization will be
+   discontinued and customers should merge away, the Bareos project will not
+   work on solving the problems with Python 3 which are caused by third-party
+   python modules.
+   If you are using the oVirt plugin, stay with Baros 22 using the Python 2
+   version of the plugin.
+
 .. index::
    pair: Plugin; oVirt
 
+
```

Comment:
I would remove anything below that warning. (for the backport we may want to add only the warning).
There's no good reason to keep that  chapter.